unix makefile fails on Parallels

cd platforms/unix
make all

EXPECT build to work

ACTUAL
c99  -o pforth pf_cglue.o pf_clib.o pf_core.o pf_inner.o pf_io.o pf_io_none.o pf_main.o pf_mem.o pf_save.o pf_text.o pf_words.o pfcompil.o pfcustom.o pf_io_posix.o pf_fileio_stdio.o  -lm
wd=$(pwd); (cd ../../fth; ${wd}/pforth -i system.fth)
/bin/sh: 1: /home/parallels/Desktop/Parallels: not found
Makefile:103: recipe for target 'pforth.dic' failed
make: *** [pforth.dic] Error 127

It looks the spaces in the folder name are a problem. Once again!
I removed the spaces from the Folder name and it worked.
I should modify the Makefile so it is immune to spaces.

--- a/platforms/unix/Makefile
+++ b/platforms/unix/Makefile
@@ -100,7 +100,7 @@ $(PFDICAPP): $(PFINCLUDES) $(PFOBJS)
 
 # Build basic dictionary image by running newly built pforth and including "system.fth".
 $(PFORTHDIC): $(PFDICAPP)
-	wd=$$(pwd); (cd $(FTHDIR); $${wd}/$(PFDICAPP) -i system.fth)
+	wd=$$(pwd); (cd $(FTHDIR); "$${wd}/$(PFDICAPP)" -i system.fth)
 	(cd $(FTHDIR); cat pforth.dic; rm -f pforth.dic) > $@
 
 $(PFDICDAT): $(PFORTHDIC) $(PFDICAPP)

